### PR TITLE
Add a description of how to test a package

### PR DESCRIPTION
**Pull request by godalming123**

### CODE REVIEW

**Comment by godalming123** [comment]
@Elara6331 can you review this

File: docs/packages/adding-packages.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 ---
 
+## How to test a package
+
+To test packages you can first create [a `lure.sh` shell file](./build-scripts.md) and then run the `lure build` comand to build the local `lure.sh` file into a install file for your OS (more info about the `build` command [here](./usage.md#build) you can then install this file to your distro and test it.
```
Comment (Elara6331) [inline docs/packages/adding-packages.md:17]:
```suggestion
To test packages you can first create [a `lure.sh` shell file](./build-scripts.md) and then run the `lure build` comand to build the local `lure.sh` file into a package for your distro (more info about the `build` command [here](./usage.md#build)). You can then install this file to your distro and test it.
```

**Review by Elara6331 — CHANGES_REQUESTED** [review]
Sorry, I've been a bit busy lately. There are some small changes I'd like you to make. Otherwise, looks good :)

**Comment by godalming123** [comment]
Will fix when I get home at 4:40

**Comment by godalming123** [comment]
@Elara6331 all of your suggested me changes have been implemented

**Review by Elara6331 — APPROVED** [review]
Thanks, merging now

**Pull request merged**